app/import.go

```diff
@@ -1699,10 +1699,12 @@ func (a *App) OldImportFile(timestamp time.Time, file io.Reader, teamId string,
 		return nil, err
 	}
 
-	img, width, height := prepareImage(data)
-	if img != nil {
-		a.generateThumbnailImage(*img, fileInfo.ThumbnailPath, width, height)
-		a.generatePreviewImage(*img, fileInfo.PreviewPath, width)
+	if fileInfo.IsImage() && fileInfo.MimeType != "image/svg+xml" {
```

Comment:
Could this ever be just `image/svg` or something else similar?

Reply:
No idea. It's a bit brittle, but it was the only example in my test Slack data archive of a common image file format that was an image but can't be thumbnailed, so I included it to reduce "noise" in the logs which is what the original ticket was complaining about.

Reply:
I'm curious about the "can't be thumbnailed" part. Do we display these types of files as images elsewhere in the product, and can't generate thumbnails on upload then? Or are they just treated as "other" files? Should we update `fileInfo.IsImage` directly?

Reply:
I looked into this a bit, and it sounds like that's just the full mime type for it. Weird that it's not `image/svg`, but I guess they wanted to future proof it in case some other format for storing SVGs ever became popular

Reply:
Whatever system we are using for creating the image thumbnails doesn't support SVG. It might be cool to have SVG thumbnails at some point, but that would be a much bigger story to implement, so for now I just blacklisted them here to reduce log noise on Slack import.